sdk/loadtestservice/azure-developer-loadtesting/pom.xml

```diff
@@ -37,8 +37,12 @@
     </developer>
   </developers>
   <properties>
+    <src.dir>src/main</src.dir>
+    <test.dir>src/test</test.dir>
```

Comment:
anything uses it?

Reply:
This is used by jacoco, is it not? Anyway this was autogenerated I guess. We can try removing it

Reply:
No, I think codegen won't do this.

Reply:
Removed